### PR TITLE
Update user guide with more user-friendly text

### PR DESCRIPTION
**Pull request by EmilyOng**

### CODE REVIEW

**Comment by EmilyOng** [comment]
Hi @JasonYapzx, is there some software that you used to edit/create the image for the UG? There are some changes that I'd like to make but don't want the styles to conflict, like the text contrast in [this](https://ay2223s1-cs2103t-w17-2.github.io/tp/images/AddCommand1.png) and also most of the images have `addressbook.json` at the bottom (need to update `preferences.json`)

File: docs/SettingUp.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ title: Setting up and getting started
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
 
-Follow the steps in the following guide precisely. Things will not work out if you deviate in some steps.
+Do be sure to follow the steps in the following guide precisely. Things may not work out if you deviate in some steps.
```
Comment (sharmaine1028) [inline docs/SettingUp.md:16]: I think we can just directly say Be sure to follow...so it sounds less formal?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -77,16 +89,16 @@ clean Graphical User Interface (GUI) for easy comprehension of expenditure and s
   <pre>
     add t/e d/Lunch a/15.60 da/15-08-2022 c/Meal
   </pre>
-    - Command : add<br>
-    - Identifier : t/e<br>
-    - Input : d/Lunch a/15.60 da/15-08-2022 c/Meal<br>
+    - Command : <pre>add</pre><br>
```
Comment (sharmaine1028) [inline docs/UserGuide.md:92]: I really like this part, it looks a lot neater now :D

File: docs/UserGuide.md [hunk 3]
```diff
@@ -61,8 +45,36 @@ clean Graphical User Interface (GUI) for easy comprehension of expenditure and s
 
 6. Refer to the [Features](#features) below for details of each command.
 
+<div markdown="span" class="alert alert-info">:information_source: **Info:**
+Congrats! You are now ready to get started on your first steps in supercharging your financial management flow!
+</div>
+
 ---
 
+## Navigating the User Guide
```
Comment (sharmaine1028) [inline docs/UserGuide.md:54]: Thanks for adding this section!

**Review by sharmaine1028 — COMMENTED** [review]
Great job!

**Comment by EmilyOng** [comment]
> Hey @EmilyOng, I just used paint to quickly put the images for tutorial earlier, was planning on changing it before consultation again on Monday! If you'd like we can start a google doc or something for each image that way we can align the styles for the UI images. But those that are put in are temporary 👍🏻

Ok I see I will update it in a docs later on, thanks!

File: src/main/resources/view/EntryListCard.fxml [hunk 4]
```diff
@@ -29,8 +29,8 @@
           </minWidth>
         </Label>
         <Label fx:id="description" styleClass="cell_big_label" text="\$first" wrapText="true" />
-            <Region prefHeight="17.0" prefWidth="17.0" HBox.hgrow="ALWAYS" />
-         <Label fx:id="amount" prefHeight="17.0" prefWidth="56.0" styleClass="cell_big_label" text="\$amount" wrapText="true" />
+            <Region HBox.hgrow="ALWAYS" />
```
Comment (EmilyOng) [inline src/main/resources/view/EntryListCard.fxml:32]: I temporarily change this to fix the issue of "amount cutting off". Will uncommit this file from the PR so we properly handle this issue in another PR.

**Comment by EmilyOng** [comment]
Hi, this is ready to be reviewed. I have updated a few images and wordings in UG.

**Comment by JasonYapzx** [comment]
> Hi, this is ready to be reviewed. I have updated a few images and wordings in UG.

Let me review it now

File: docs/UserGuide.md [hunk 5]
```diff
@@ -29,87 +12,124 @@ helping students to manage their budget. It is **optimised for use via Command L
 clean Graphical User Interface (GUI) for easy comprehension of expenditure and savings. _Interested?_ Head over to
 “Quick Start” to get started. Enjoy!
 
+<div markdown="span" class="alert alert-info">:information_source: **Info:**
+Already an expert? Jump right straight to the [features](#features) section to see how you can get the most out of PennyWise!
+</div>
+
```
Comment (JasonYapzx) [inline docs/UserGuide.md:18]: good idea to add this, thannks!

File: docs/UserGuide.md [hunk 6]
```diff
@@ -120,127 +140,221 @@ clean Graphical User Interface (GUI) for easy comprehension of expenditure and s
...
+|---------------------|----------------------------------------------------------------------------------------------|
+| **`t/ENTRY_TYPE`**  | An entry type refers to either `e` for "Expenditure" or `i` for "Income"                     |
+| **`d/DESCRIPTION`** | Description for the entry that you are adding                                                |
+| **`a/AMOUNT`**      | Amount of the entry, formatted to 2 decimal places                                           |
+| **`da/DATE`**       | Date where the entry is added in `dd-mm-YYYY` format, e.g. `01-01-2022` for 1st January 2022 |
+| **`c/CATEGORY`**    | [Category](#categorising-your-expenses-and-income) that the entry belongs to                 |
+
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 The default date is the current date on your computer!
 </div>
```
Comment (JasonYapzx) [inline docs/UserGuide.md:199]: I forgot to remove this, I don't think our currently implementation has this feature for the default date.

File: docs/UserGuide.md [hunk 7]
```diff
@@ -120,127 +140,221 @@ clean Graphical User Interface (GUI) for easy comprehension of expenditure and s
...
+
+1. View a LineGraph of all expenditures in a specified month 
+   - Examples: `view t/e mo/2022-01` where we specify the month to be January 2022.
+   - Expected: `Show graphically all expenditure by month` and a LineGraph on the right of the application
+
+   ![ViewCommandExpenditureMonth](images/ug/ViewCommandExpenditureMonth.png)
+   
+2. View a LineGraph of all incomes in a specified month
+   * Examples: `view t/i mo/2022-01` <br>
    Expected: `Show graphically all income by month` and a LineGraph on the right of the application
```
Comment (JasonYapzx) [inline docs/UserGuide.md:342]: small nit to pick, since everything is changed to a '-', we should follow the format for these 2 lines as well

**Review by JasonYapzx — APPROVED** [review]
LGTM just some small changes to fix!

**Review by JasonYapzx — APPROVED** [review]
LGTM just some small changes to fix!

For the `DeleteCommandExpenditure.png` and `DeleteCommandIncome.png` files are the transparent Kungfu panda and SpaceX stocks meant to be in there? I thought it was a diagram bug initially haha. 
image

**Comment by EmilyOng** [comment]
> For the `DeleteCommandExpenditure.png` and `DeleteCommandIncome.png` files are the transparent Kungfu panda and 
SpaceX stocks meant to be in there? I thought it was a diagram bug initially haha.

Oh yea haha I wanted to show the deleting effect. Perhaps I can just remove those transparent pics to make it simpler

**Comment by JasonYapzx** [comment]
> > For the `DeleteCommandExpenditure.png` and `DeleteCommandIncome.png` files are the transparent Kungfu panda and
> > SpaceX stocks meant to be in there? I thought it was a diagram bug initially haha.
> 
> Oh yea haha I wanted to show the deleting effect. Perhaps I can just remove those transparent pics to make it simpler

Would something like this be a bit clearer? haha looking at how they scruntize our UI bugs in dryrun, i think in the actual PE they might think this is a UG bug as well

I added this to the bottom of the Excalidraw page
image

**Comment by EmilyOng** [comment]
Done, thanks!

**Pull request merged**